app/components/global/AppBar/AppBar.js

```diff
@@ -46,24 +46,28 @@ const DesktopView = styled(Flex)`
   `};
 `
 
-const AppBar = ({ user }) => (
-  <AppBarContainer>
-    <BurgerMenu menuItems={user && [{ label: 'Dashboard', link: '/' }]} />
-    <LogoLink href="https://elifesciences.org">
-      <img alt="eLife" src="/assets/elife-logo.png" />
-    </LogoLink>
-    <DesktopView>
-      <Separator />
-      <Box flex="1 1 auto">
-        {user && (
-          <AppBarLink exact to="/">
-            Dashboard
-          </AppBarLink>
-        )}
-      </Box>
-    </DesktopView>
-    <ProfileMenu user={user} />
-  </AppBarContainer>
-)
+const AppBar = ({ user, userMenuItems, defaultMenuItems }) => {
+  const menuItems = [...(user ? userMenuItems : []), ...defaultMenuItems]
```

Comment:
It's confusing to check for the existence of `user` before we use `userMenuItems` given the parameters we're passing in. Instead we can pass in both within a single object as the first parameter, i.e.
```
const AppBar = ({ user, defaultMenuItems }) => {
  const { currentUser, userMenuItems = [] } = user
  const menuItems = [...userMenuItems, ...defaultMenuItems]
  ...
}
```